boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -399,4 +409,147 @@ TEST_F(
     }
 }
 
+// DCM-8
+TEST_F(
+    DcmStateMachineTest,
+    regen_allowed_only_when_going_faster_than_5kph_and_airs_closed)
+{
+    SetInitialState(App_GetDriveState());
```

Comment:
You should make use of 
```
std::nextafter(YOUR_THRESHOLD_VALUE, std::numeric_limits<float>::max());
```
Instead of setting the overflow wheel speed to 5.1

Reply:
ooh cool